### PR TITLE
Setup web chat

### PR DESCRIPTION
**Pull request by dharmendrak**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-6342
https://tools.hmcts.net/jira/browse/ROC-6341

### Change description

Setup web chat

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/app.ts [hunk 1]
```diff
@@ -61,6 +61,8 @@ app.use(cookieEncrypter(config.get('secrets.cmc.encryptionKey'), {
   }
 }))
 
+// Web Chat
+app.use('/webchat', express.static(path.join(__dirname, '/node_modules/@hmcts/ctsc-web-chat/assets')))
```
Comment (robertparkinson) [inline src/main/app.ts]: If you have copied everything into our folders in the above gulp file you shouldn't need to point to the node_modules folder. It should all be part of our files?

File: src/main/views/macro/support.njk [hunk 2]
```diff
@@ -1,20 +1,41 @@
 {% from "externalLink.njk" import externalLink %}
 
+{% macro _webChatWidget(label = 'Web chat') %}
+      <h2 class="heading-medium" style="margin-bottom:5px; margin-top: 10px;">{{ t(label)}}</h2>
+      <div id="ctsc-web-chat"></div>
+{% endmacro%}
+
+{% macro _emailWidget(email = 'moneyclaims@justice.gov.uk') %}
```
Comment (robertparkinson) [inline src/main/views/macro/support.njk]: this email should be retrieved from the CONTACT_EMAIL variable

File: src/main/app.ts (hunk 1 above)
Comment (sabahirfan) [reply src/main/app.ts]: Shouldn't we use from the library itself rather than copying locally? We don't want to end up different versions in our local.

Comment (robertparkinson) [reply src/main/app.ts]: every time you do a yarn install then do a gulp it should copy and over-ride the files to the correct folders. We may need to add in a script to clear out the old ones. You normally only get problems if they delete files from their repo and we don't clear it up with gulp

File: src/main/views/macro/support.njk (hunk 2 above)
Comment (dharmendrak) [reply src/main/views/macro/support.njk]: done

File: src/main/app.ts (hunk 1 above)
Comment (sabahirfan) [reply src/main/app.ts]: 👍

**Pull request merged**